### DISCUSSION TITLE
can't connect to ejbca-ce admin UI

### DISCUSSION THREAD

**Discussion by srnovh** [Keyfactor/ejbca-ce#562] [General]
Hi,
We have an issue in our UI admin web, when we try to connect to admin Ui : https://our-url/ejbca/adminweb
we have this error message and we can't connect to our admin :
**_An exception has occurred.
Index 4 out of bounds for length 2_** 

Any tips to help? This seems to be coming from our database, but how can we fix it?

**Comment by primetomas** [comment]
Check the server.log for details of the error.
https://doc.primekey.com/ejbca/troubleshooting-guide
